modules/runtime/pages/process-manager-application.adoc

```diff
@@ -0,0 +1,61 @@
+= Creating a process manager application
+:description: This page explains how to create your own custom process manager application
+
+[NOTE]
+====
+
+For Subscription editions only. +
```

Comment:
⚠️ this is not the wording we use in other pages. We always reference Teamwork, ..., Enterprise.
Please check with @NathalieC 
Not using consistent messaging is a pain for readers and will make maintenance harder. Please see https://github.com/bonitasoft/bonita-documentation-site/issues/279

Reply:
Thank you very much for your comment. I'll look into it

Reply:
Hello. I have no statistics to provide on this topic, but a quick search brings back a little bit of both.
Here is why:
- When we have started with such notes, this was because we needed to explain a feature was in some editions but not all, like Efficiency and Performance, but no Teamwork, or only in Performance, and not in Teamwork of Efficiency.
- When we have started to sell Enterprise editions and no more TW, Eff, Perf, the _all inclusive_ "For Subscriptions editions only" has come up as a handy way to express "teamwork, efficiency, performance, enterprise" as there was no need to differentiate between all 4, and it started to become looong to write. 
- But at the same time, on some pages, we have continued to write all 5 subscription editions.

**True** that we need to clarify.

So, my take on this is that:
- when a feature is about all Subscription editions, we should be short and say "For Subscription editions only".  I find it more effective, and I think this is a clear enough vocabulary for a user of any subscription edition.

So **I am taking this item for me to check and replace** when needed.

- when a feature is for only some subscription editions but not all, we should write the details.

@tbouffard how does that sound to you?

Reply:
OK this makes sense.
But I am just afraid that introducing this new wording without explanation will make this harder to understand. Remember there is no explanation in the documentation nor on the website about the extra edition names.

Here is the list of `Editions` names we now have in the documentation:
- Community
- Teamwork
- Efficiency
- Performance
- Enterprise
- Subscription (the one you refer to)

About changing the other pages, I suggest we **decide how** to do it directly in https://github.com/bonitasoft/bonita-documentation-site/issues/279 and continue the discussion there. We have ways to reduce duplicating the message.

Reply:
Notice there are places where we will continue to use the old editions, and this will probably the sole pages where the Teamwork name will be used 😿 

Like in https://documentation.bonitasoft.com/bonita/2021.2/identity/organization-management-in-bonita-bpm-studio#_profile_management

![image](https://user-images.githubusercontent.com/27200110/150173585-f4d91eb6-fff8-46f7-8e11-a3c49a29163d.png)